explanation/architecture.rst

```diff
@@ -138,6 +138,15 @@ The storage role encapsulates the software defined storage component of the clou
 
 Nodes with the storage role must have free, un-partitioned disks for use by Ceph.
 
+Network
+^^^^^^^
+
+The network role encapsulates the networking component of the cloud; this is provided by MicroOVN which provides a simple and distributed solutions for networking.
+.. note::
```

Comment:
Design considerations page should have a note note on both Compute and Network nodes provide OVN L3 gateway services. In the future only Network node provide OVN L3 gateway service.